tests/components/mfi/test_sensor.py

```diff
@@ -134,7 +135,7 @@ async def test_uom_temp(port, sensor):
     """Test the UOM temperature."""
     port.tag = "temperature"
     assert sensor.unit_of_measurement == TEMP_CELSIUS
-    assert sensor.device_class == DEVICE_CLASS_TEMPERATURE
+    assert sensor.device_class == SensorDeviceClass.TEMPERATURE
```

Comment:
You could use `is` here to check the device class

Reply:
Agreed, done.